.github/workflows/examples-linux-imx.yaml

```diff
@@ -55,3 +55,18 @@ jobs:
               run: |
                   ./scripts/examples/imxlinux_example.sh \
                   examples/chip-tool examples/chip-tool/out/aarch64
+            - name: Build thermostat
+              timeout-minutes: 30
+              run: |
+                  ./scripts/build/build_examples.py \
```

Comment:
need to use the `scripts/run_in_build_env.sh` wrapper script to execute these, so that build environment is set up.

Reply:
> need to use the `scripts/run_in_build_env.sh` wrapper script to execute these, so that build environment is set up.

Thank you for the kind reminder.